wslpy/system.py

```diff
@@ -77,5 +90,16 @@ def shellEnvVar(input):
     return __shellEnvVar__(input)
 
 
+def RegEnvVarList():
```

Comment:
Rename this to sysEnvVarList, and update the documentation

Reply:
Done